fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountReadPlatformServiceImpl.java

```diff
@@ -126,7 +141,7 @@ public SavingsAccountReadPlatformServiceImpl(final PlatformSecurityContext conte
             final ChargeReadPlatformService chargeReadPlatformService,
             final EntityDatatableChecksReadService entityDatatableChecksReadService, final ColumnValidator columnValidator,
             final SavingsAccountAssembler savingAccountAssembler, PaginationHelper paginationHelper,
-            DatabaseSpecificSQLGenerator sqlGenerator) {
+            DatabaseSpecificSQLGenerator sqlGenerator, SavingsAccountTransactionRepository savingsAccountTransactionsRepository) {
```

Comment:
Could've replced this with a lombok annotation.

Reply:
Yes. But I am afraid if it would break the existing implementation

Reply:
This is a constructor. If it breaks the existing impl, you'll know it because the startup of the Spring context will simply fail.